ci/build.py

```diff
@@ -37,11 +41,54 @@
 import platform
 from copy import deepcopy
 from itertools import chain
-from subprocess import call, check_call
+from subprocess import call, check_call, check_output
 from typing import *
 from util import *
+try:
+    pip_install('docker')
+    import docker
+    import docker.models
+    import docker.errors
+    import signal
+    import atexit
+except ImportError as e:
```

Comment:
A usual import error I think should be enough

Reply:
⚡